test/project_types/theme/commands/generate/env_test.rb

```diff
@@ -0,0 +1,189 @@
+# frozen_string_literal: true
+require 'project_types/theme/test_helper'
+
+module Theme
+  module Commands
+    module GenerateTests
+      class EnvTest < MiniTest::Test
+        resp = [200,
+                { "themes" =>
+                 [{ "id" => 2468,
+                    "name" => "my_theme" },
+                  { "id" => 1357,
+                    "name" => "your_theme" }] }]
+        THEMES = resp[1]['themes'].map { |theme| [theme['name'], theme['id']] }.to_h
+
+        CONFIG = { "development" =>
+                   { "password" => "boop",
+                     "theme_id" => "2468",
+                     "store" => "shop.myshopify.com" } }
+
+        def test_prompts_for_credentials
+          FakeFS do
+            File.write('config.yml', CONFIG.to_yaml)
+            context = ShopifyCli::Context.new
+
+            CLI::UI::Prompt.expects(:ask)
+              .with(context.message('theme.generate.env.ask_store_default', 'shop.myshopify.com'))
+              .returns('shop.myshopify.com')
```

Comment:
We should use different values for this test to make sure that we override the defaults if they're given.